src/__test__/cancelWorkflow.test.ts

```diff
@@ -0,0 +1,43 @@
+import { cancelWorkflow } from "@/controllers/cancelWorkflow";
```

Comment:
These tests look great - nice and short while also testing exactly what needs to be tested